std/ascii.d

```diff
@@ -250,7 +250,7 @@ bool isAlpha(dchar c) @safe pure nothrow @nogc
   +/
 bool isLower(dchar c) @safe pure nothrow @nogc
 {
-    return c >= 'a' && c <= 'z';
+    return c - 'a' < 26;
```

Comment:
For instance this is equally clear but it looks tidier.

Reply:
The disadvantage of the new version is, that one has to look up the definition of `c` to find out if only the letters are meant or all ascii symbols up to 'z'.

Reply:
> The disadvantage of the new version is, that one has to look up the definition of `c` to find out if only the letters are meant or all ascii symbols up to 'z'.

Commenting would be a solution for more unreadable logic.